Description: Remove unavailable features
 As we use Debian packaged libzstd instead of building the bundled,
 such features are not enabled at build time and unavailable.
Last-Update: 2022-09-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -48,18 +48,11 @@
 [features]
 arrays = []
 bindgen = []
-debug = ["zstd-sys/debug"]
 default = [
-    "legacy",
     "arrays",
     "zdict_builder",
 ]
 doc-cfg = []
 experimental = ["zstd-sys/experimental"]
-legacy = ["zstd-sys/legacy"]
-no_asm = ["zstd-sys/no_asm"]
 pkg-config = []
-std = ["zstd-sys/std"]
-thin = ["zstd-sys/thin"]
 zdict_builder = ["zstd-sys/zdict_builder"]
-zstdmt = ["zstd-sys/zstdmt"]
